TLC5955.cpp

```diff
@@ -204,21 +204,14 @@ void TLC5955::updateLeds()
   } else {
     analogWrite(_gsclk, 1);
   }
+
   if (enforce_max_current)
   {
-    double power_output_amps = ((double)power_output_counts / (double)UINT16_MAX) * LED_CURRENT_AMPS;
+    double power_output_amps = getTotalCurrent();
     if (power_output_amps > max_current_amps)
-    {
-      Serial.print(F("Current output ("));
-      Serial.print(power_output_amps);
-      Serial.print(F(") exceeds maximum current output ("));
-      Serial.print(max_current_amps);
-	  Serial.println(')');
-      return;
-    }
+      return 1;
   }
```

Comment:
```suggestion
  } else if (power_output_amps == 0.) {  # 0. comparity check is OK since we know all currents are positive
    analogWrite(_gsclk, 0);
  } else {
    analogWrite(_gsclk, 1);
  }
```

Please reuse the code that you refactored....

Reply:
fixed